GeneralsMD/Code/Libraries/Source/WWVegas/CMakeLists.txt

```diff
@@ -4,16 +4,11 @@ add_library(z_wwcommon INTERFACE)
 target_compile_definitions(z_wwcommon INTERFACE
     #NOMINMAX
     WIN32_LEAN_AND_MEAN
-    $<$<CONFIG:Debug>:WWDEBUG>
-    $<$<CONFIG:Debug>:_DEBUG>
-)
-
-target_compile_options(z_wwcommon INTERFACE
-    ${GENZH_FLAGS}
 )
 
 target_link_libraries(z_wwcommon INTERFACE
     d3d8lib
+    gz_config
```

Comment:
Will all targets we build inherit this config or is there a chance that we missed one?

Reply:
Its also included in GameEngine so anything that was getting the debug macros previously should be getting them still. Its possible that some aren't, but if that is the case they probably weren't before this PR either and need identifying to include this target.